frontend/src/pages/PointsPage/components/PointDisplay/index.tsx

```diff
@@ -29,6 +29,9 @@ export function PointDisplay(props: PointProps): JSX.Element {
                       <Typography variant='h5' component='h2'>
                         {event.name}
                       </Typography>
+                      <Typography color='textSecondary'>
+                        {event.type} event
+                      </Typography>
                       <Typography className={classes.pos} color='textSecondary'>
                         {format(parseISO(attendance.startTime), 'PP')}
```

Comment:
Idk if there is another suitable color for the attendance date but imo it's better to have the color be at least a bit different from the event type. Thoughts?